composable.js

```diff
@@ -126,6 +126,12 @@
         toFloat: function (item) {
             return isString(item) || isNumber(item) ? parseFloat(item) : null;
         },
+        toFixed: function (item) {
```

Comment:
I would expect `precision` to be the static argument and the number to be the dynamic argument, which I think is the opposite of this change, but maybe I'm misunderstanding the use-case. Are you doing `'toFixed:2'` or `'toFixed:12.345'`?